manifests/cf-manifest/manifest/050-rds-broker.yml

```diff
@@ -67,9 +99,40 @@ jobs:
                 supportUrl: "https://forums.aws.amazon.com/forum.jspa?forumID=60"
               plan_updateable: true
               plans:
+                - id: "b7d0a368-ac92-4eff-9b8d-ab4ba45bed0e"
+                  name: "S-dedicated-9.5"
+                  description: "20GB Storage, Dedicated Instance, Max 200 Concurrent Connections. Postgres Version 9.5, db.t2.small"
```

Comment:
The name of the DB Instance Class (`db.t2.small`) appears alongside the Postgres version. I think this is confusing. The structure of the name `db.t2.small` will make sense to people who already know what AWS DB Instance Classes are, but is jargon in a weird context for anybody else.

Suggested edit:
```
Postgres Version 9.5. DB Instance Class db.t2.small.
```

This labels the value [the same way as AWS refers to it](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html).